pkg/httpserver/log.go

```diff
@@ -60,6 +60,7 @@ func logResponses(log *zap.Logger, h http.Handler) http.Handler {
 				// we are deliberately not logging the request URI as it has
 				// sensitive information in it.
 				zap.Int("code", code),
+				zap.String("request-id", rw.Header().Get("X-Storj-Request-Id")),
```

Comment:
this might be better as `GetReqID(ctx)` then you don't have to hardcode the header name